### PR TITLE
Add servables

### PR DESCRIPTION
**Pull request by trthhrtz**

### CODE REVIEW

File: hydrosdk/model.py [hunk 1]
```diff
@@ -151,7 +154,9 @@ def __upload(self, cluster):
             "runtime": {"name": self.runtime.name,
                         "tag": self.runtime.tag,
                         "sha256": self.runtime.sha256},
-            "contract": contract_to_dict(self.contract)
+            "contract": contract_to_dict(self.contract),
+            "installCommand": self.install_command,
+            "metadata": self.metadata
```
Comment (KineticCookie) [inline hydrosdk/model.py]: server expects `metadata` keys and values to be of type `str`
Need to add check/conversion before uploading it

File: hydrosdk/servable.py [hunk 2]
```diff
@@ -1,39 +1,50 @@
...
     pass
 
 
 class Servable:
     BASE_URL = "/api/v2/servable"
 
-    @staticmethod
-    def find(servable_name):
-        pass
+    def get(self, servable_name):
```
Comment (KineticCookie) [inline hydrosdk/servable.py]: Why get isn't static?

File: hydrosdk/servable.py [hunk 3]
```diff
@@ -1,39 +1,50 @@
...
+    #     res = self.cluster.request("GET", self.BASE_URL)
+    #     if res.ok:
+    #         return res.json()
+    #     else:
+    #         return None
 
-    @staticmethod
-    def list_for_model(model_name, model_version):
-        pass
+    def create(self, model_name, model_version):
```
Comment (KineticCookie) [inline hydrosdk/servable.py]: Why it is not static?

File: hydrosdk/servable.py [hunk 4]
```diff
@@ -48,6 +59,9 @@ def predictor(self, secure=False, shadowed=True):
 
         return predictor
 
+    def list(self):
```
Comment (KineticCookie) [inline hydrosdk/servable.py]: static?

File: hydrosdk/servable.py [hunk 5]
```diff
@@ -61,3 +75,9 @@ def logs(self, follow=False):
         else:
             raise ServableException(resp)
 
+    def delete(self, servable_name):
```
Comment (KineticCookie) [inline hydrosdk/servable.py]: static?

File: hydrosdk/servable.py [hunk 6]
```diff
@@ -1,39 +1,50 @@
 from urllib.parse import urljoin
+
 import sseclient
+
 from .predictor import GRPCPredictor, ShadowlessGRPCPredictor
```
Comment (KineticCookie) [inline hydrosdk/servable.py]: If it's not used, I guess it's better to delete them for now.

File: hydrosdk/model.py (hunk 1 above)
Comment (trthhrtz) [reply hydrosdk/model.py]: Fixed

File: hydrosdk/servable.py (hunk 2 above)
Comment (trthhrtz) [reply hydrosdk/servable.py]: Fixed

File: hydrosdk/servable.py (hunk 3 above)
Comment (trthhrtz) [reply hydrosdk/servable.py]: Fixed

File: hydrosdk/servable.py (hunk 4 above)
Comment (trthhrtz) [reply hydrosdk/servable.py]: Fixed

File: hydrosdk/servable.py (hunk 5 above)
Comment (trthhrtz) [reply hydrosdk/servable.py]: Fixed

File: hydrosdk/servable.py (hunk 6 above)
Comment (trthhrtz) [reply hydrosdk/servable.py]: Do you mean imports `from .predictor import GRPCPredictor, ShadowlessGRPCPredictor` or `def predictor?`

Comment (trthhrtz) [reply hydrosdk/servable.py]: Fixed

**Pull request merged**